### PR TITLE
tutorial and Journal week 3

### PR DESCRIPTION
**Pull request by rosalsm**

### CODE REVIEW

**Comment by jeffworley** [comment]
Rosa, this looks like a great start!  I always enjoy reading your personal narrative and reflection and think that your approach of tackling the tutorial as you learn it could have some benefits in your retention.  You miss, get a :+1:

**Comment by nickycadavillo** [comment]
:+1: Great start!  I like your idea of looking at the big picture with CSS - definitely good advice.

**Comment by ShayHall** [comment]
Good start.  My problem is that I'm afraid what I learn each week _will_ stay in that week and not in my head.  Keeping shining. :sunny:

File: tutorial-week-3.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+I would like to do my tutorial about HTML. At the same time that I am learning I can write so that way I will review 
+whatever I learn new everyday this week.
+Today was basics like elements in HTML. 
+How to start:
+<!DOCTYPE html> were we inform the the browser that we are using the last version of html and the begining of the document.
+<head> will be the next element and it identifies the top of the document including the metadata and it may include a document title.
```
Comment (kellymurray) [inline tutorial-week-3.md:7]: Good idea. You could do a tutorial about how to start an HTML page (maybe an `index.html` page?).

**Pull request merged**